packages/components/src/components/domainproperties/samples/SampleTypeDesigner.tsx

```diff
@@ -0,0 +1,394 @@
+        const newModel = model.merge({parentAliases: newAliases}) as SampleTypeModel;
+        this.setState(() => ({model: newModel}));
+    };
+
+    addParentAlias = (id:string, newAlias: IParentAlias): void => {
+        const {model} = this.state;
+        let {parentAliases} = model;
+        const newModel = model.merge({parentAliases:parentAliases.set(id, newAlias)}) as SampleTypeModel;
+        this.setState(() => ({model: newModel}));
+    };
+
+    removeParentAlias = (id:string) => {
+        const {model} = this.state;
+        let {parentAliases} = model;
+        const aliases = parentAliases.delete(id);
+        const newModel = model.set('parentAliases', aliases) as SampleTypeModel;
+        this.setState(() => ({model: newModel}));
+    };
+
+    renderDetailsPanel = () => {
+        const {noun = 'Sample Type', nameExpressionInfoUrl, nameExpressionPlaceholder } = this.props;
+        const {model, parentOptions,} = this.state;
+        return (
+            <>
+                <SampleTypePropertiesPanel
+                    model={model}
+                    parentOptions={parentOptions}
+                    onParentAliasChange={this.parentAliasChange}
+                    onAddParentAlias={this.addParentAlias}
+                    onRemoveParentAlias={this.removeParentAlias}
+                    noun={noun}
+                    nameExpressionInfoUrl={nameExpressionInfoUrl}
+                    nameExpressionPlaceholder={nameExpressionPlaceholder}
+                    updateModel={this.onFieldChange}
+                />
+            </>
+        )
+    };
+
+    domainChangeHandler = (newDomain: DomainDesign): void => {
+        const {defaultSampleFieldConfig} = this.props;
+        let {model} = this.state;
+        let invalidDomainField = undefined;
+        if (newDomain && newDomain.fields) {
+            newDomain.fields.forEach(field => {
+                if (field && field.name && field.name.toLowerCase() === defaultSampleFieldConfig.name.toLowerCase()) {
+                    invalidDomainField = field.name
+                }
+            });
+        }
+
+        this._dirty = true;
+        model = model.merge({domain:newDomain}) as SampleTypeModel;
+        this.setState(() => ({
+            model,
+            invalidDomainField
+        }));
+    };
+
+    renderDomainPanel = () => {
+        const {noun, domainDesignerStickyHeight} = this.props;
+        const {model} = this.state;
+        const {domain} = model;
+
+        return (
+            <>
+                {domain &&
+                <DomainForm
+                        domain={domain}
+                        onChange={this.domainChangeHandler}
+                        showHeader={true}
+                        initCollapsed={true}
+                        collapsible={true}
+                        helpNoun={noun.toLowerCase()}
+                        useTheme={false}
+                        appPropertiesOnly={true}
```

Comment:
Shouldn't the useTheme and appPropertiesOnly values be coming in as props to this and passed through here? I assume the false and true values are for the SM use case, but wouldn't the LKS page want them to be the opposite?